tutorials/katacoda/thanos/2-lts/step3.md

```diff
@@ -0,0 +1,64 @@
+# Step 3 - Fetching metrics from Bucket
+
+In this step, we will learn about Thanos Store Gateway and how to deploy it.
+
+## Thanos Components
+
+Let's take a look at all the Thanos commands:
+
+```docker run --rm quay.io/thanos/thanos:v0.16.0 --help```{{execute}}
+
+You should see multiple commands that solve different purposes, block storage based long-term storage for Prometheus.
+
+In this step we will focus on thanos `store gateway`:
+
+```
+  store [<flags>]
+    Store node giving access to blocks in a bucket provider
+```
+
+## Store Gateway:
+
+* This component implements the Store API on top of historical data in an object storage bucket. It acts primarily as an API gateway and therefore does not need
+significant amounts of local disk space.
+* It keeps a small amount of information about all remote blocks on the local disk and keeps it in sync with the bucket.
+This data is generally safe to delete across restarts at the cost of increased startup times.
+
+You can read more about [Store](https://thanos.io/tip/components/store.md/) here.
+
+### Deploying store for "EU1" Prometheus data
+
+```
+docker run -d --net=host --rm \
+    -v /root/editor/bucket_storage.yaml:/etc/thanos/minio-bucket.yaml \
+    --name store-gateway \
+    quay.io/thanos/thanos:v0.16.0 \
+    store \
+    --objstore.config-file /etc/thanos/minio-bucket.yaml \
+    --http-address 0.0.0.0:19091 \
+    --grpc-address 0.0.0.0:19191
+```{{execute}}
+
+## How to query Thanos store data?
+
+In this step, we will see how we can query Thanos store data which has access to historical data from the `thanos` bucket, and let's play with this setup a bit.
+
+Click on the [Querier UI `Graph` page](https://[[HOST_SUBDOMAIN]]-29090-[[KATACODA_HOST]].environments.katacoda.com/graph) and try querying data for a year or two by inserting metrics [k8s_app_metric0](https://[[HOST_SUBDOMAIN]]-29090-[[KATACODA_HOST]].environments.katacoda.com/graph?g0.range_input=2d&g0.end_input=2019-10-18%2011%3A26&g0.max_source_resolution=0s&g0.expr=k8s_app_metric0&g0.tab=0). Make sure `deduplication` is selected and you will be able to discover all the data fetched by Thanos store.
+
+![](https://github.com/soniasingla/thanos/raw/master/tutorials/katacoda/thanos/2-lts/query.png)
+
+Also, you can check all the active endpoints located by thanos-store by clicking on [Stores](https://[[HOST_SUBDOMAIN]]-29090-[[KATACODA_HOST]].environments.katacoda.com/stores).
```

Comment:
We need a maybe another question of "How to check if you query data from bucket or from Prometheus.

And we can click to NewUI -> Enable store filter -> Ask to pick only store gateway so it will be visible only store GW data is used